packages/angularjs/src/services/model.ts

```diff
@@ -200,6 +201,11 @@ export class Model extends ModelBase {
 
         const that = this
         this.initialize = _.once(this.initialize || function defaultInitializer() {
+            // Begin Watching if already completed
```

Comment:
You could listen to the change event to avoid another watcher.  For instance: `collection.on('change', console.log)`.

Reply:
`collection.on('change', ...)` set each individual model to `model.changed=true/false`? That was the main point of enabling the watcher as they are used as normal Models with their tracking.

Originally I had to do a hacky thing of for eaching through collection.models and manually set model.watcher()

Reply:
How it works is each time a model watcher detects a change, it triggers the collection's event as well.

You can see it trigger on this line:

https://github.com/Sitetheory/stratus/blob/master/packages/angularjs/src/services/model.ts#L297

My solution is if you change that line to:

```ts
this.collection.throttleTrigger('change', this)
```

Then you can do this in the collection:

```ts
this.on('change', (model: Model) => {
  if (!model.changed) {
    return
  }
  console.log('child model changed:', model.patch)
})
```

Reply:
are you asking to not set the models to be watched though? (your first comment is still confusing me).
model will never be set to changed if it isn't watched from my tests and i had to manually enable the watch.

(`collection.on('change', ...)` in my use cases still won't do use as I only use the collection just for an initial fetch and to hold any array)

Reply:
That's alright.  If what you have works fine, I'll keep that going and take note for a possible future assessment.  Thanks!

Reply:
Hahaha... I'm just still trying to figure out what the goal of your suggestion was.